This is a testharness.js-based test.
PASS [document] unsafe-none => unsafe-none
FAIL [document] unsafe-none => credentialless assert_equals: expected "error" but got "retrieved"
PASS [document] unsafe-none => credentialless + CORP
PASS [document] unsafe-none => require-corp
PASS [document] unsafe-none => require-corp + CORP
PASS [document] credentialless => unsafe-none
PASS [document] credentialless => credentialless
PASS [document] credentialless => require-corp
PASS [document] credentialless => require-corp + CORP
PASS [document] require_corp => unsafe-none
PASS [document] require_corp => credentialless
PASS [document] require_corp => require-corp
PASS [dedicated_worker] unsafe-none => unsafe-none
FAIL [dedicated_worker] unsafe-none => credentialless assert_equals: expected "error" but got "retrieved"
PASS [dedicated_worker] unsafe-none => credentialless + CORP
FAIL [dedicated_worker] unsafe-none => require-corp assert_equals: expected "error" but got "retrieved"
PASS [dedicated_worker] unsafe-none => require-corp + CORP
PASS [dedicated_worker] credentialless => unsafe-none
PASS [dedicated_worker] credentialless => credentialless
FAIL [dedicated_worker] credentialless => require-corp assert_equals: expected "error" but got "retrieved"
PASS [dedicated_worker] credentialless => require-corp + CORP
PASS [dedicated_worker] require_corp => unsafe-none
PASS [dedicated_worker] require_corp => credentialless
PASS [dedicated_worker] require_corp => require-corp
PASS [shared_worker] unsafe-none => unsafe-none
FAIL [shared_worker] unsafe-none => credentialless assert_equals: expected "error" but got "retrieved"
PASS [shared_worker] unsafe-none => credentialless + CORP
FAIL [shared_worker] unsafe-none => require-corp assert_equals: expected "error" but got "retrieved"
PASS [shared_worker] unsafe-none => require-corp + CORP
PASS [shared_worker] credentialless => unsafe-none
PASS [shared_worker] credentialless => credentialless
FAIL [shared_worker] credentialless => require-corp assert_equals: expected "error" but got "retrieved"
PASS [shared_worker] credentialless => require-corp + CORP
PASS [shared_worker] require_corp => unsafe-none
PASS [shared_worker] require_corp => credentialless
PASS [shared_worker] require_corp => require-corp
PASS [service_worker] unsafe-none => unsafe-none
FAIL [service_worker] unsafe-none => credentialless assert_equals: expected "error" but got "retrieved"
PASS [service_worker] unsafe-none => credentialless + CORP
PASS [service_worker] unsafe-none => require-corp
PASS [service_worker] unsafe-none => require-corp + CORP
PASS [service_worker] credentialless => unsafe-none
PASS [service_worker] credentialless => credentialless
PASS [service_worker] credentialless => require-corp
PASS [service_worker] credentialless => require-corp + CORP
PASS [service_worker] require_corp => unsafe-none
PASS [service_worker] require_corp => credentialless
PASS [service_worker] require_corp => require-corp
Harness: the test ran to completion.

